LOC7386Abaraj.xml

```diff
@@ -0,0 +1,56 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LOC7386Abaraj" xml:lang="en" type="place">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>አባራጅ፡</title>
```

Comment:
I guess, that refers to all places, that I encoded, that they shall get a transliterated name in their title. I will do in a minute. 
Which transliteration do you prefer in this case? ʾAbārāǧ;  ʾAbbārāǧǧ or something else. It might be named after an esteemed person and for that reason contains ʾAbbā

Reply:
I don't know either, this is always an issue. A choice has to be made (variants can also be given), and if a reader has a better hypothesis they can correct it. But it is important that the information is also accessible to people who cannot read fidal.